sdk/core/core-rest-pipeline/src/nodeHttpClient.ts

```diff
@@ -300,7 +300,7 @@ function streamToText(stream: NodeJS.ReadableStream): Promise<string> {
   });
 }
 
-function getBodyLength(body: RequestBodyType): number | null {
+export function getBodyLength(body: RequestBodyType): number | null {
```

Comment:
if we export I think we have to mark it as internal in a doc comment to avoid the documentation system picking it up, right @deyaaeldeen  ?

Reply:
Added the `@internal` tag to the method